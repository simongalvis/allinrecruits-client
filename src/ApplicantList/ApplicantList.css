.applicantList{
    display: flex;
}
.applicantTile{
    display: flex;
    flex-direction: column;
    font-size: 17px;
    height:auto
}
.applicantList{
    
    flex-direction: column-reverse;
}
p{
    display: inline;
}
#delete-btn{
    margin: 2% 35% ;
    display: flex;
    font-size: 17px;
    height:auto;
    text-align: center;
    justify-content: center;
    width:auto;
    border: 1px solid black;
    border-radius: 20px;
    background-color:rgb(252, 85, 85);
    font-weight: bold;
    
}
@media only screen and (min-width: 777px) {
    .applicantTile{
        width:500px;
    }
    .tiles-container{
        display: flex;
        justify-content: center;
        align-items: center;
    } 
  }