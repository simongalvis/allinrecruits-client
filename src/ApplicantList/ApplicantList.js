import React from 'react';
import './ApplicantList.css'
import NavBar from '../NavBar/NavBar'

class ApplicantList extends React.Component{

    render(){
        return(
            <div className="ApplicantList">
                <NavBar/>
                <h1>ApplicantList</h1>
                <ul id="applicant-tiles">
                    <li className="applicant-tile"><b>Bob</b></li>
                    <li className="applicant-tile"><b>Mary</b></li>
                    <li className="applicant-tile"><b>John</b></li>
                    <li className="applicant-tile"><b>Martha</b></li>
                    <li className="applicant-tile"><b>Sandra</b></li>
                    <li className="applicant-tile"><b>Francois</b></li>


                </ul>
            </div>
        )
    }
}
export default ApplicantList;