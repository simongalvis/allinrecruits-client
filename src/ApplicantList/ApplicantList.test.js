import React from 'react'
import ReactDOM from 'react-dom'
import ApplicantList from './ApplicantList'
import renderer from 'react-test-renderer';
import {BrowserRouter} from 'react-router-dom'

describe(`ApplicantList component`, () =>{
    it(`renders without crashing`, () =>{
        const div = document.createElement('div');

        ReactDOM.render(<BrowserRouter><ApplicantList/></BrowserRouter>, div)

        ReactDOM.unmountComponentAtNode(div);
    })
    it(`Renders UI as expected`, () =>{
        const tree = renderer
            .create(<BrowserRouter><ApplicantList/></BrowserRouter>)
            .toJSON();
        expect(tree).toMatchSnapshot();
    })
})