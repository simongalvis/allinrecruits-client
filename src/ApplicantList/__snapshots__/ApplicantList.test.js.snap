// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ApplicantList component Renders UI as expected 1`] = `
<div
  className="ApplicantList"
>
  <div
    className="NavBar"
  >
    <header>
      <a
        href="/"
        onClick={[Function]}
      >
        <img
          alt="all-in-education-logo"
          id="main-logo"
          src="all-in-logo.png"
        />
      </a>
      <a
        href="/admin-login"
      >
        <button
          id="admin-login-btn"
        >
          Admin Login
        </button>
      </a>
    </header>
  </div>
  <h1>
    Submissions: 
  </h1>
  <div
    className="tiles-container"
  >
    <ul
      id="applicant-tiles"
    />
  </div>
</div>
`;
